contributing/docker.md

```diff
@@ -1,4 +1,4 @@
-# Server Development
+# Base Adapter Images
```

Comment:
We need to explain why someone should read this page. It's not clear to me when someone would need this information.